### PR TITLE
feat(vercel): Test for deployed binaries

### LINKED ISSUE #1617: Vercel: Confirm binary deployed to Vercel Functions
Is it correct one? Only one?

### PR DESCRIPTION
**Pull request by janpio**
- Test deployed binaries on Vercel as well
- Also rename `vercel-api` to `vercel-cli`

closes #1617

### CODE REVIEW

File: platforms-serverless/netlify-ci/functions/index.js [hunk 1]
```diff
@@ -33,6 +33,12 @@ exports.handler = async function (event, context, callback) {
 
   const deleteManyUsers = await client.user.deleteMany()
 
+  /*
+  const dirTree = require("directory-tree");
+  const tree = dirTree(process.env.LAMBDA_TASK_ROOT);
+  console.dir(tree, { depth: null });
+  */
+
```
Comment (Jolg42) [inline platforms-serverless/netlify-ci/functions/index.js:41]: Do we want to keep these commented lines around?

**Review by Jolg42 — APPROVED** [review]
Looks good to me! (maybe remove commented code)

**Review by millsp — APPROVED** [review]

File: platforms-serverless/netlify-ci/functions/index.js (hunk 1 above)
Comment (janpio) [reply platforms-serverless/netlify-ci/functions/index.js:41]: Simplest way to get an overview if our actual tests start to break (when Vercel or AWS Lambda changes the structure in some way) - so I considered this helpful.

**Comment by janpio** [comment]
Hm, merging this first caused this to be added to the other PR :/

**Comment by Jolg42** [comment]
Oh 💡 didn't know! Do you want me to revert this / merge other / merge here?

**Comment by janpio** [comment]
No I think it's fine - I just renamed the other PR to also include this one :D

**Pull request merged**